### DISCUSSION TITLE
Corners of the input images

### DISCUSSION THREAD

**Discussion by nicolasloizeau** [OpenStitching/stitching#241] [Q&A]
Is there some easy way to get the positions of the 4 corners of each input image on the final panorama ?
Even just in the particular case of the AffineStitcher, that would be useful.

**Comment by lukasalexanderweber** [comment]
You can use the homography from the obtained cameras to calculate the location of the corners in the final result:

https://stackoverflow.com/questions/76003489/how-to-get-image-coordinates-after-successful-match-and-stitch-using-opencv
